Model/Ui/AdyenBoletoConfigProvider.php

```diff
@@ -81,7 +81,7 @@ public function getConfig()
             'payment' => [
                 self::CODE => [
                     'isActive' => true,
-                    'redirectUrl' => $this->_urlBuilder->getUrl(
+                    'successpage' => $this->_urlBuilder->getUrl(
```

Comment:
is there a reason why you went for the `successpage` instead of `successPage`?

Reply:
not a good reason :). I will change it to keep the naming convention

Reply:
Could @acampos1916 @cyattilakiss check again ?